app/assets/javascripts/admin/line_items/controllers/line_items_controller.js.coffee

```diff
@@ -69,12 +69,12 @@ angular.module("admin.lineItems").controller 'LineItemsCtrl', ($scope, $timeout,
         $scope.bulk_order_form.$setPristine()
         $scope.saving = false
       ).catch ->
-        alert "Some errors must be resolved be before you can update orders.\nAny fields with red borders contain errors."
+        alert "#{t("unsaved_changes_warning")}"
```

Comment:
Can this be simplified to just `t("unsaved_changes_warning")` without `"#{...}"`?